epix/clustering.py

```diff
@@ -268,7 +269,7 @@ def classify(types, parenttype, creaproc, edproc):
 
     # If our data does not match any classification make it a nest None type
     # TODO: fix me
-    return 0, 0, 12
+    return np.iinfo(np.int16).max, np.iinfo(np.int16).max, 12
```

Comment:
```suggestion
    return infinity, infinity, 12
```

Reply:
Changed in: https://github.com/XENONnT/epix/pull/30/commits/eb7d44fd053a8331d5018eb070bf0a927de67988